assets/php/class-wgpb-block-grid-base.php

```diff
@@ -172,11 +172,10 @@ protected function set_categories_query_args( &$query_args ) {
 	 * @return array List of product IDs
 	 */
 	protected function get_products() {
-		// @todo Cache this
 		if ( 'product-top-rated' === $this->block_name ) {
-			add_filter( 'posts_clauses', array( __CLASS__, 'order_by_rating_post_clauses' ) );
+			add_filter( 'posts_clauses', array( WC()->query, 'order_by_rating_post_clauses' ) );
```

Comment:
@ryelle This fixes the issue I mentioned where `order_by_rating_post_clauses` doesn't exist in this class. I will review again at some point to see if these are actually needed anymore but not in this PR.